src/components/AvatarGroup/AvatarGroupCounterTooltipContentVirtualizedList.tsx

```diff
@@ -12,7 +13,7 @@ const LIST_OPTIONS = Object.freeze({
 });
 
 export type AvatarItem = {
-  value: AvatarProps & { tooltipContent: string | ReactElement | ReactElement[] };
+  value: AvatarProps & { tooltipContent: ElementContent | ElementContent[] };
```

Comment:
I am merging it because no one reviewed it and it is a bit urgent (blocker for a team to update a version after waiting long time) - please review V2 pr: https://github.com/mondaycom/monday-ui-react-core/pull/1318
If something is needed to be fix, I will also fix it for this version.